### PR TITLE
Add MINFRACT to swhs and correct the constraints for v_p.

### PR DESCRIPTION
**Pull request by halonazhao**

### CODE REVIEW

**Comment by szymczdm** [comment]
Hi @halonazhao, can you please not leave the description for your PR's blank? It's a lot easier to tell what you're doing if you include a bit more detail than just the title.

File: code/drasil-example/Drasil/SWHS/Unitals.hs [hunk 1]
```diff
@@ -446,6 +447,8 @@ tank_length_max = mkDataDef (unitary "tank_length_max"
   (nounPhraseSP "maximum length of tank")
   (sub (eqSymb tank_length) (Atomic "max")) metre Rational) 50
 
+frac_min_aux    = mkDataDef frac_min $ (dbl 1.0) * (10 $^ (negate 6))
```
Comment (szymczdm) [inline code/drasil-example/Drasil/SWHS/Unitals.hs]: We should probably keep this in mind for #941 as we don't want scientific notation hacks.

Comment (halonazhao) [reply code/drasil-example/Drasil/SWHS/Unitals.hs]: Ok. After scientific notation is done. I will check and correct all hacks.

**Comment by halonazhao** [comment]
Alright. Next time, I'll try to be more specific.

**Comment by JacquesCarette** [comment]
Unfortunately, there's another problem with this. As far as I can tell, the LaTeX ends up doing $MINFRAC$ (or the equivalent thereof), which ends up looking awful.  One should never display words in math notation.

Roughly speaking, the 'bug' is that `Atomic` should really have just a `Char` and not a `String`. [So the problem is much bigger than just this one example, but this one really makes it visible].  One potential hack would be that all `Atomic` of length >1 should be displayed (in LaTeX) using \mathsf. That would fix all problems at once.

@halonazhao can you file an issue about this please?

**Pull request merged**